nntrainer/models/neuralnet.cpp

```diff
@@ -852,6 +868,23 @@ int NeuralNetwork::addLayer(NodeType layer) {
   return status;
 }
 
+NeuralNetwork &NeuralNetwork::copyConfiguration(NeuralNetwork &from) {
+  if (this != &from) {
+    model_props = from.model_props;
+    model_flex_props = from.model_flex_props;
+    loss = from.loss;
+    opt = from.opt;
+
+    NetworkGraph f_graph = from.getNetworkGraph();
+    for (auto &l_node : f_graph.getLayerNodes()) {
+      auto new_node = l_node->cloneConfiguration();
+      std::shared_ptr<ml::train::Layer> n = std::move(new_node);
+      addLayer(n);
```

Comment:
How about this?
```suggestion
      addLayer(std::move(new_node));
```

Reply:
Thanks. but this needs to type conversion. I updated the code accordingly.